docs/_includes/chips-combobox.html

```diff
@@ -0,0 +1,396 @@
+<div id="chips-combobox">
+    {% include section-header.html name="chips-combobox" version=page.versions.chips-combobox %}
+
+    <p>Chips Combobox are creators and managers of <span class="highlight">chip</span> components. They allow user search of existing chip options, additions, and removals of chips. The search/selection follows the pattern of the combobox pattern.</p>
+
+    <h3>Empty Chips Combobox</h3>
+    <p>This is the simplest type of implementation with an empty chips combobox component usually encountered on initial information entry.</p>
+
+    <p><strong>NOTE</strong>: For empty Chips Combobox implementations, to avoid layout issues, make sure <span class="highlight">.chips-combobox__items</span> does NOT have any whitespace.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="chips-combobox">
+                <div class="chips-combobox__items" role="group" aria-describedby="chips-combobox-1-heading"></div>
```

Comment:
`chips-combobox-1-heading` doesn't seem to resolve to an id. Do we need aria-describedby and role=group here?

Reply:
### aria-describedby
I thought I had commented on the heading IDs in the PR somewhere, but I lost track of where myself. 😄 

One of the PR feedbacks was to remove labels, helper text, and error messaging. When removing the label, I thought leaving the IDs in would be helpful to indicate for ebay ui to create an option in the component for teams to pass this ID in and have the combobox items linked to the label external to the module.  If it's more confusing in isolation than helpful, I can remove it. 

### role="group"
Don't we need to have the element containing all the chips be a group to indicate a group of `chip` items?

Reply:
A few options:

1. No ARIA. Remove group and aria-describedby and leverage a simple unordered list (with no label/description) and hope that user can deduce the relationship to the combobox below it.
2. ARIA. Keep group and add `aria-label="Selected sports"` or a more generic `aria-label="Selected values". We could also add `aria-roledescription="Chips"` if we think chips might be a user interface term that users are familiar with.
3. A mix. Use unordered list (no role=group) and apply`aria-label="Selected sports"` to the ul